Scientific documentation for CCPP-SCM v7.0.0 (#1077)
NCAR/ccpp-physics: physics/docs/pdftxt/all_schemes_list.txt

\b Radiation

>  - \subpage GFS_RRTMG_page 

Reviewer: Reminder that links to Tech Doc need to be updated

 - \subpage GFS_NOAH 
 - \subpage RUCLSM 
 - \subpage NoahMP 

>  - \subpage CLM_LAKE_model

Reviewer: Seems strange to list lake under land. I think lake should be its own category

\b Cumulus \b Parameterizations
 - \subpage GFS_SAMFdeep 

>  - \subpage cellular_automata

Reviewer: Since CA does not function for SCM, I suggest removing from the list of supported suites

 - \subpage GFS_UGWP_v0 
 - \subpage GFS_UNIFIED_UGWP
 - \subpage GFS_drag_suite

>  - \subpage GFS_ugwpv1_gsldrag

Reviewer: We do not call it gsl anymore

Author: It is the subroutine name. The word follows  "subpage" does not show in the website.

 - \subpage GFS_SFCSICE 

\b Others

>  - \subpage GFS_SPP

Reviewer: Does SPP work for SCM? If not, need to remove

\section allsuite_overview Physics Suites

> The publicly supported host model that use CCPP v7.0.0 is the UFS Weather Model and the CCPP Single Column Model (CCPP-SCM). 

Reviewer: ```suggestion
The only publicly supported host model for CCPP v7.0.0 is the CCPP Single Column Model (CCPP-SCM). 
```
